lib/Config/Hooks.php

```diff
@@ -365,4 +366,19 @@ public function setupPostNotifications() {
       10, 3
     );
   }
+
+  public function setupFooter() {
+    $this->wp->addFilter(
+      'admin_footer_text',
+      [$this, 'setFooter'],
+      1, 1
+    );
+  }
+
+  public function setFooter($text) {
+    if (!Menu::isOnMailPoetAdminPage()) {
+      return $text;
+    }
+    return '<a href="https://feedback.mailpoet.com/" target="_blank">Give feedback</a>';
```

Comment:
I guess we should add also `noopener noreferrer`, right?

When at it, do you think adding the check for `Menu::isOnMailPoetAdminPage()` directly to `setupFooter` is safe or could there be any problem? Just thinking we wouldn't need to register the hook on non-MP pages at all.

Reply:
you are right, I'll make the changes. 

- I didn't add  `noopener noreferrer` because the target is our domain so it is not really necessary here
- the code is inspired by woocommerce that does it this way as well, but I can move the check up, no problem.